Formula/retdec.rb

```diff
@@ -0,0 +1,52 @@
+class Retdec < Formula
+  desc "Retargetable machine-code decompiler based on LLVM"
+  homepage "https://github.com/avast/retdec"
+  license all_of: ["MIT", "Zlib"]
+  head "https://github.com/avast/retdec.git", branch: "master"
+
+  stable do
+    url "https://github.com/avast/retdec.git",
+      branch:   "master",
+      revision: "8503282c6d1295ba58ed5fed4ccb5605ed3febfd"
+    version "5.0"
+  end
```

Comment:
It should be using tagged versions

Reply:
They only have tagged versions for releases (Which would require patches for OpenSSL)

Reply:
So we need to wait for a release that includes this. People have lived without retdec in Homebrew/core for a while. They can wait a little longer.

Reply:
Okay, let's keep this pull request open until new release then?